pwalvarado/qeue_example.rb

```diff
@@ -0,0 +1,22 @@
+require 'thread'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.